Exercise08answers.R

```diff
@@ -0,0 +1,39 @@
+#Exercise 08 Answers
+
+#Question 1
+#Creating a function
+top<-function(r,line)
+  + {subf<-r[1:line,]
+  + return(subf)}
+#test using random numbers
+top(iris,3)
+
+#Question 2
+#Last 2 rows and last two columns
+iris[149:150,4:5]
+
+#number of observations for each species
+unique(iris$Species) #3 different species
+a<-subset(iris,Species=="setosa")
+length(a[,5])
+
+b<-subset(iris,Species=="versicolor")
+length(b[,5])
+
+c<-subset(iris,Species=="virginica")
+length(c[,5])
+
+#Rows with Sepal.Width>3.5
+SW<-subset(iris,Sepal.Width>3.5)
+dim(SW) #number of rows
+

```

Comment:
If you follow the lecture, we used [] to extract data and not using subset function here

Reply:
-0.25

Reply:
I have a further discussion with Dr Jones and he said this will be Ok
+0.25